src/lib/csv-export/index.ts

```diff
@@ -0,0 +1,73 @@
+import {ModelPluginInterface} from '../../interfaces';
+import {ModelParams} from '../../types';
+
+import * as path from 'path';
+import * as fs from 'fs/promises';
+
+import {ERRORS} from '../../util/errors';
+import {buildErrorMessage} from '../../util/helpers';
+const {InputValidationError} = ERRORS;
+
```

Comment:
general note: i think after this PR is merged we should have a PR in the IF repo to add a sample/test impl for the CSV export model. I can help with that if needed.